Cargo.toml

```diff
@@ -48,6 +49,7 @@ url = "2.2.2"
 wasi-outbound-http = { path = "crates/outbound-http" }
 wasmtime = "0.35.3"
 
+
```

Comment:
Super pedantic: I don't think the extra line is needed?

Reply:
👌 Of course, I'll delete this blank line